javascript/array-methods-2/js-array-methods-2_animal-strings/index.js

```diff
@@ -39,18 +40,18 @@ const indexOfGiraffe = null;
 // Hint: There is no need to upper-/lowercase the strings before sorting them.
 // Hint: sort() mutates the original array, which is bad.
 // -> Use animals.slice().sort(...) to make a copy.
-const animalsSortedAlphabetically = null;
+const animalsSortedAlphabetically = animalStrings.slice().sort();
```

Comment:
I thought the exercise explicitly wanted us to use the conditions... I was unsure if we could use the simple version `.sort()`. But it also works, so I don't see a problem with that :)